BookReader/BookReader.js

```diff
@@ -752,6 +757,16 @@ BookReader.prototype.setupKeyListeners = function() {
                         self.right();
                     }
                     break;
+                case KEY_MINUS:
+                case KEY_NUMPAD_SUBTRACT:
+                    e.preventDefault();
+                    self.zoom(-1);
+                    break;
+                case KEY_EQUAL:
+                case KEY_NUMPAD_ADD:
+                    e.preventDefault();
+                    self.zoom(+1);
```

Comment:
The `+1` is not necessary here. Could just be `1`.

Reply:
Opting to leave this as is; I think the plus makes this easier to read. Seeing `zoom(+` makes me think that it's zooming in. Seeing `zoom(1)` doesn't necessarily do that; it looks like it might be resetting the zoom to 100%. (longer term we might create some wrapper function called `zoomIn` and `zoomOut` that hide the abstraction).